File: DrivingAcademy/DrivingAcademy/Entities/Student.cs
using System.ComponentModel.DataAnnotations;

namespace DrivingAcademy.Entities
{
    public partial class Student
    {
        public int Id { get; set; }
        [Required]
        [Range(18, 70, ErrorMessage = "La edad debe ser entre 18 y 70 años")]
        public short Age { get; set; }
        [Required]
        public string Name { get; set; } = null!;
        [Required]
        public string Identification { get; set; } = null!;
        public int TypeLicenceId { get; set; }
        public Licence TypeLicence { get; set; } = null!;
    }
}

File: driving-academy/src/app/models/Student.ts
export interface Student {
  StudentId: number;
  Name: string;
}

export interface Lesson {
  Id: number;
  Name: string;
  Module: string;
}

export interface DetailStudent {
  StudentId: number;
  Name: string;
  TypeLicence: string;
  Identification: string;
  Age: number;
  Lessons: Lesson[];
}
